### PR TITLE
Create ui-c8y-1021-23-0-ui-as-a-alarms-widget-user-i-want-to-filter-alarms.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-60797](https://cumulocity.atlassian.net/browse/MTM-60797)
Original PR: [7755](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7755)

[MTM-60797]: https://cumulocity.atlassian.net/browse/MTM-60797?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1021-23-0-ui-as-a-alarms-widget-user-i-want-to-filter-alarms.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: UI as a alarms widget user i want to filter alarms by date
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-23-0-ui-as-a-alarms-widget-user-i-want-to-filter-alarms.md]: The Claude AI suggested to use the following title:
```suggestion
title: Alarms widget allows filtering alarms by date
```

File: content/change-logs/application-enablement/ui-c8y-1021-23-0-ui-as-a-alarms-widget-user-i-want-to-filter-alarms.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60797
+version: 1021.23.0
+---
+UI as a alarms widget user i want to filter alarms by date
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-23-0-ui-as-a-alarms-widget-user-i-want-to-filter-alarms.md]: The Claude AI suggested to use the following body:
```suggestion
The alarms widget in the Cumulocity IoT user interface previously displayed all alarms without the ability to filter them by date. With this change, users can now filter the displayed alarms in the widget by specifying a date range. This allows users to narrow down the shown alarms to a specific time period they are interested in, making it easier to find relevant alarms and improving the overall user experience when working with alarms in the Cumulocity IoT application.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1021-23-0-ui-as-a-alarms-widget-user-i-want-to-filter-alarms.md (hunk 1 above)
Comment (eniosultan) [inline content/change-logs/application-enablement/ui-c8y-1021-23-0-ui-as-a-alarms-widget-user-i-want-to-filter-alarms.md]:
```suggestion
title: Alarms list widget can now use date range filtering.
```

File: content/change-logs/application-enablement/ui-c8y-1021-23-0-ui-as-a-alarms-widget-user-i-want-to-filter-alarms.md (hunk 2 above)
Comment (eniosultan) [inline content/change-logs/application-enablement/ui-c8y-1021-23-0-ui-as-a-alarms-widget-user-i-want-to-filter-alarms.md]:
```suggestion
The alarms list widget in the Cumulocity IoT user interface previously displayed all alarms without the ability to filter them by date. With this change, users can now filter the displayed alarms in the widget by specifying a date range. This allows users to narrow down the shown alarms to a specific time period they are interested in. Users can select the date range either specifically for the widget or bind the widget to the dashboard time context.
```

**Review by eniosultan — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1021-23-0-ui-as-a-alarms-widget-user-i-want-to-filter-alarms.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60797
+version: 1021.23.0
+---
+The alarms list widget in the Cumulocity IoT user interface previously displayed all alarms without the ability to filter them by date. With this change, users can now filter the displayed alarms in the widget by specifying a date range. This allows users to narrow down the shown alarms to a specific time period they are interested in. Users can select the date range either specifically for the widget or bind the widget to the dashboard time context.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-23-0-ui-as-a-alarms-widget-user-i-want-to-filter-alarms.md]:
```suggestion
The "Alarm list" widget previously displayed all alarms without the ability to filter them by date. With this change, users can now filter the displayed alarms in the widget by specifying a date range. This allows users to narrow down the shown alarms to a specific time period they are interested in. Users can select the date range either specifically for the widget or bind the widget to the dashboard time context.
```

File: content/change-logs/application-enablement/ui-c8y-1021-23-0-ui-as-a-alarms-widget-user-i-want-to-filter-alarms.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Alarms list widget can now use date range filtering.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-23-0-ui-as-a-alarms-widget-user-i-want-to-filter-alarms.md]:
```suggestion
title: Alarms list widget now offers date range filtering
```

**Review by BeateRixen — APPROVED** [review]

**Review by rahulc8y — APPROVED** [review]

**Pull request merged**